docs/delivery-manual/trial/Trial.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Trial
+sidebar_position: 1
+---
+
+## Overview
+In the Trial Phase, you'll build and test an MVP, engage with trial participants, issue and/or verify credentials, and measure performance. This phase is crucial for validating your digital trust solution in a real-world context before broader implementation.
+
+## Key outcomes
+
+1.  **Preparation and MVP Build**: Develop your issuer and verifier services with support from the Digital Trust team. Build and prepare an MVP for the trial.
+    
+2.  **Participant engagement**:  
+    Identify and recruit potential participants for the trial, informing them about the process and expectations.
+    
+3.  **Issuance and verification**:  
+    Issue digital credentials to participants and verify these credentials, allowing participants to experience the full cycle of the digital trust solution.
+
+4.  **Performance evalutation**: 
+    Collect and analyze performance metrics and user feedback to determine the trial's success and inform decisions about scaling.
+
+## Core Activities
+Core Activities are key tasks to complete during the Align phase, either independently or with Digital Trust team support, to prepare for adoption.
+
+### Build and test
+==Adopter led initiative==
```

Comment:
Highlight markdown doesn't work on docusaurus. I think we're italicizing and placing on next line.

Reply:
See above